schemas/client.sql

```diff
@@ -140,13 +140,13 @@ CREATE TABLE BlahdRecords (
   VORoleID                    INT          NOT NULL, -- foreign key
   CEID                        INT          NOT NULL, -- foreign key
   GlobalJobId                 VARCHAR(255) DEFAULT NULL,
-  LrmsId                      VARCHAR(255) DEFAULT NULL,
+  LrmsId                      VARCHAR(255) DEFAULT '',
```

Comment:
Do we want to make an update script for clients? As we've done for [server schema changes](https://github.com/apel/apel/blob/1.9.0-1/scripts/update-1.7.0-1.8.2.sql) in the past?

Reply:
I've made a note in https://github.com/apel/apel/issues/243